apps/api/jupyter-requirements.txt

```diff
@@ -45,6 +45,7 @@ googleapis-common-protos==1.62.0
 db-dtypes==1.2.0
 fastparquet==2024.2.0
 oracledb==2.2.0
+pymssql==2.3.1
```

Comment:
I've seen `pymssql` is `LGPL` licensed. I think it would probably be fine to use it, but I'd rather pick something MIT/BSD/Apache licensed just to be sure. Can we use another lib here?

Reply:
@AlexandreSenpai [pyodbc](https://docs.sqlalchemy.org/en/20/dialects/mssql.html#module-sqlalchemy.dialects.mssql.pyodbc) seems promising.

Reply:
This is actually fine.